openmetadata-docs/content/v1.4.x/developers/contribute/build-code-and-run-tests/openmetadata-server.md

```diff
@@ -10,13 +10,13 @@ Learn how to run the OpenMetadata server in development mode by using Docker and
 - For an easy installation of MySQL and ES, just install Docker on your local machine and run the following commands from the top-level directory
 
 ```shell
-docker compose -f docker/development/docker-compose.yml up mysql elasticsearch --build -d
+docker compose -f docker/development/docker-compose.yml up mysql opensearch --build -d
```

Comment:
The line here, it is for mysql startup, we have elasticsearch here and not opensearch

Reply:
@ayush-shah updated it